fluss-lakehouse/fluss-lakehouse-paimon/src/main/java/com/alibaba/fluss/lakehouse/paimon/source/split/HybridSnapshotLogSplitState.java

```diff
@@ -26,10 +26,13 @@ public class HybridSnapshotLogSplitState extends SourceSplitState {
     private boolean snapshotFinished;
 
     /** The next log offset to read. */
-    private long offset;
+    private long currentOffset;
```

Comment:
Maybe `nextOffset` to be align with the comment? The `currentOffset` may confuse users this the offset that just read to.